mlir/test/Target/LLVMIR/tune-cpu.mlir

```diff
@@ -0,0 +1,7 @@
+// RUN: mlir-translate -mlir-to-llvmir %s | FileCheck %s
```

Comment:
This may require x86-registered-target as well?

Reply:
Thanks again!  Yes, you are absolutely correct.